### PR TITLE
[Bulk] Validation Fixes AM 5-16 (resolved comments)

### PR DESCRIPTION
**Pull request by AqMoh**
This PR fixes ~50 issues across 6 articles.

### CODE REVIEW

File: aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc4/intro-to-aspnet-mvc-4.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc4/intro-to-aspnet-mvc-4
 title: "Intro to ASP.NET MVC 4 | Microsoft Docs"
 author: Rick-Anderson
-description: "An updated version if this tutorial is available here using Visual Studio 2013. The new tutorial uses ASP.NET MVC 5, which provides many improvements over t..."
+description: "An updated version if this tutorial is available here using Visual Studio 2013. The new tutorial uses ASP.NET MVC 5, which provides users many improvements over t..."
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc4/intro-to-aspnet-mvc-4.md:5]: Let's fix the grammar error on the first sentence and cut off the second sentence so it's not truncated.  Like this, "An updated version **of** this tutorial is available here using Visual Studio 2013. The new tutorial uses ASP.NET MVC 5."

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application
 title: "Advanced Entity Framework Scenarios for an MVC Web Application (10 of 10) | Microsoft Docs"
 author: tdykstra
-description: "The Contoso University sample web application demonstrates how to create ASP.NET MVC 4 applications using the Entity Framework 5 Code First and Visual Studio..."
+description: "The Contoso University sample web application explains how to create ASP.NET MVC 4 applications using the Entity Framework 5 Code First and Visual Studio..."
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md:5]: Let's get rid of the extra periods at the end of the second sentence.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md [hunk 3]
```diff
@@ -31,11 +31,11 @@ In the previous tutorial you implemented the repository and unit of work pattern
 
 For most of these topics, you'll work with pages that you already created. To use raw SQL to do bulk updates you'll create a new page that updates the number of credits of all courses in the database:
 
-![Update_Course_Credits_initial_page](advanced-entity-framework-scenarios-for-an-mvc-web-application/_static/image1.png)
+![Update_Course_Credits_initial_new_page](advanced-entity-framework-scenarios-for-an-mvc-web-application/_static/image1.png)
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md:34]: Let's replace this with alt-text.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md [hunk 4]
```diff
@@ -31,11 +31,11 @@ In the previous tutorial you implemented the repository and unit of work pattern
...
 
 And to use a no-tracking query you'll add new validation logic to the Department Edit page:
 
-![Department_Edit_page_with_duplicate_administrator_error_message](advanced-entity-framework-scenarios-for-an-mvc-web-application/_static/image2.png)
+![Contoso_University_Department_Edit_page_with_duplicate_administrator_error_message](advanced-entity-framework-scenarios-for-an-mvc-web-application/_static/image2.png)
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md:38]: Let's replace this with alt-text.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md [hunk 5]
```diff
@@ -109,7 +109,7 @@ This method will be used for both `HttpGet` and `HttpPost`. When the `HttpGet` `
 
 When the **Update** button is clicked and the `HttpPost` method runs, `multiplier` will have the value entered in the text box. The code then calls the repository `UpdateCourseCredits` method, which returns the number of affected rows, and that value is stored in the `ViewBag` object. When the view receives the number of affected rows in the `ViewBag` object, it displays that number instead of the text box and submit button, as shown in the following illustration:
 
-![Update_Course_Credits_rows_affected_page](advanced-entity-framework-scenarios-for-an-mvc-web-application/_static/image6.png)
+![Contoso_University_Update_Course_Credits_rows_affected_page](advanced-entity-framework-scenarios-for-an-mvc-web-application/_static/image6.png)
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/advanced-entity-framework-scenarios-for-an-mvc-web-application.md:112]: Let's replace this with alt-text.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/creating-a-more-complex-data-model-for-an-asp-net-mvc-application.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/creating-a-more-complex-data-model-for-an-asp-net-mvc-application
 title: "Creating a More Complex Data Model for an ASP.NET MVC Application (4 of 10) | Microsoft Docs"
 author: tdykstra
-description: "The Contoso University sample web application demonstrates how to create ASP.NET MVC 4 applications using the Entity Framework 5 Code First and Visual Studio..."
+description: "The Contoso University sample web application demonstrates how to create ASP.NET MVC 4 applications by using the Entity Framework 5 Code First and Visual Studio..."
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/creating-a-more-complex-data-model-for-an-asp-net-mvc-application.md:5]: Let's get rid of the additional periods at the end.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application
 title: "Implementing Basic CRUD Functionality with the Entity Framework in ASP.NET MVC Application (2 of 10) | Microsoft Docs"
 author: tdykstra
-description: "The Contoso University sample web application demonstrates how to create ASP.NET MVC 4 applications using the Entity Framework 5 Code First and Visual Studio..."
+description: "The Contoso University sample web application tutorial demonstrates how to create ASP.NET MVC 4 applications using the Entity Framework 5 Code First and Visual Studio..."
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md:5]: Let's get rid of the additional periods at the end of the 2nd sentence.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md [hunk 8]
```diff
@@ -26,11 +26,11 @@ In the previous tutorial you created an MVC application that stores and displays
 
 In this tutorial, you'll create the following web pages:
 
-![Student_Details_page](implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application/_static/image1.png)
+![Contoso_University_Student_Details_page](implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application/_static/image1.png)
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md:29]: Let's use alt-text here.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md [hunk 9]
```diff
@@ -26,11 +26,11 @@ In the previous tutorial you created an MVC application that stores and displays
...
 
-![Student_Edit_page](implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application/_static/image2.png)
+![Contoso_University_Student_Edit_page](implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application/_static/image2.png)
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md:31]: Let's use alt-text here.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md [hunk 10]
```diff
@@ -26,11 +26,11 @@ In the previous tutorial you created an MVC application that stores and displays
...
 
-![Student_Create_page](implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application/_static/image3.png)
+![Contoso_University_Student_Create_page](implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application/_static/image3.png)
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md:33]: Let's use alt-text here.

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md [hunk 11]
```diff
@@ -67,8 +67,7 @@ The scaffolded code for the Students `Index` page left out the `Enrollments` pro
     The `ValidateAntiForgeryToken` attribute helps prevent [cross-site request forgery](../../security/xsrfcsrf-prevention-in-aspnet-mvc-and-web-pages.md) attacks.
 
 <a id="overpost"></a>
-
-    > [!WARNING]
+ ```> [!WARNING]
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md:70]: Should the 3 backticks be on their own line?

File: aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-inheritance-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 12]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-inheritance-with-the-entity-framework-in-an-asp-net-mvc-application
 title: "Implementing Inheritance with the Entity Framework in an ASP.NET MVC Application (8 of 10) | Microsoft Docs"
 author: tdykstra
-description: "The Contoso University sample web application demonstrates how to create ASP.NET MVC 4 applications using the Entity Framework 5 Code First and Visual Studio..."
+description: "The Contoso University sample web application demonstrates to users how to create ASP.NET MVC 4 applications using the Entity Framework 5 Code First and Visual Studio..."
```
Comment (RedZephyr13) [inline aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-inheritance-with-the-entity-framework-in-an-asp-net-mvc-application.md:5]: Let's get rid of additional periods.

**Comment by AqMoh** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request merged**